nautobot_circuit_maintenance/migrations/0014_migrate_notes.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.16 on 2024-09-13 08:49
+
+from django.db import migrations
+
+
+def move_data_to_new_model(apps, schema_editor):
+    CircuitMaintenance = apps.get_model("nautobot_circuit_maintenance", "CircuitMaintenance")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    OldModel = apps.get_model("nautobot_circuit_maintenance", "Note")
+    NewModel = apps.get_model("extras", "Note")
+
+    for old_obj in OldModel.objects.all():
+        NewModel.objects.create(
+            note=old_obj.title + "\n" + old_obj.comment,
```

Comment:
Given Notes support Markdown, I think a single `\n` is mostly ignored in the rendering. Perhaps?

```suggestion
            note=f"### {old_obj.title}\n\n{old_obj.comment}",
```

Maybe a similar change in the generation of new notes in `handler.py` as well?

Reply:
thanks

Reply:
@chadell planning to address this?

Reply:
yup, I forgot about it